router/functionHandler.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	"github.com/fission/fission"
 	poolmgrClient "github.com/fission/fission/poolmgr/client"
+	"github.com/gorilla/mux"
```

Comment:
@life1347 I tried goimports, but it did not sort the imports. Could you provide your toolchain configs or add it to the `hack` folder to automatically make the check ?